src/main/index.js

```diff
@@ -26,18 +27,12 @@ process.on('exit', function () {
 
 // Ensure CONFIG_PATH exists.
 mkdirp.sync(config.CONFIG_PATH)
-/* *CONFIG* */
-let argv = sliceArgv(process.argv)
 
 if (config.IS_PRODUCTION) {
   // When Electron is running in production mode (packaged app), then run React
   // in production mode too.
   process.env.NODE_ENV = 'production'
```

Comment:
@Jikstra I wonder if this is needed or even matters. Do you know if `NODE_ENV` means anything to react when in production?

Reply:
The more we can get rid of small complexities/oddities like this, the better. This code doesn't say anything about _why_ this is important.

Reply:
> @Jikstra I wonder if this is needed or even matters. Do you know if `NODE_ENV` means anything to react when in production?

Would be suprised about that, as far as i know the process global isn't defined in browser js. Let's remove and see, can add it back if we need to. We aren't using NODE_ENV anyway so who cares.

Reply:
What I'm actually getting at is if we can remove `config.IS_PRODUCTION` :)